db/schema.rb

```diff
@@ -10,8 +10,73 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 0) do
+ActiveRecord::Schema[7.1].define(version: 2024_10_03_193757) do
```

Comment:
My best advice for you - never ever change anything in schema.rb manually (I'm not talking about this case, of course). Even if you have conflicts, be super careful! Otherwise you'll be in a huge trouble like I was yesterday )) By the way, this line of code show the name of last migration that was performed. You can find file in Migration folder with exact this name and see what migration it was.

Reply:
Thanks for the tips!